# Description of objects in scene graph

ObjectProperty[] properties                                # A list of scene object properties